src/components/IssueStats/IssueStats.tsx

```diff
@@ -1,122 +1,102 @@
-    issuer?: ActivityByIdReturnType | null;
     estimate?: Date | null;
     estimateType?: DateType | null;
-    priority?: string | null;
+    priority?: Priority | null;
     achivedCriteriaWeight?: number | null;
     hasPrivateDeps?: boolean;
-    mode?: 'compact' | 'default';
     onCommentsClick?: () => void;
 }
 
-const StyledIssueStats = styled(Text)<Pick<IssueStatsProps, 'mode'>>`
-    display: flex;
-    align-items: center;
-
-    ${({ mode }) =>
-        mode === 'default' &&
-        `
-            padding-left: ${gapXs};
-        `}
-`;
-
-const StyledDotSep = styled.span`
-    display: flex;
-    align-items: center;
-`;
-
-const DotSep: React.FC<{ children: React.ReactNode }> = ({ children }) => (
-    <StyledDotSep>
-        <Dot />
+const Separator: React.FC<{ children: React.ReactNode }> = ({ children }) => (
+    <>
+        <div className={s.Separator} />
         {children}
-    </StyledDotSep>
+    </>
 );
 
 export const IssueStats: React.FC<IssueStatsProps> = ({
-    issuer,
     owner,
     estimate,
     estimateType,
     priority,
     comments,
     achivedCriteriaWeight,
-    updatedAt,
     hasPrivateDeps,
-    mode,
     onCommentsClick,
 }) => {
-    const locale = useLocale();
-
-    const issuers = useMemo(() => {
-        if (issuer && owner && owner.id === issuer.id) {
-            return [owner];
-        }
-
-        return [issuer, owner].filter(Boolean) as NonNullable<ActivityByIdReturnType>[];
-    }, [issuer, owner]);
-
     return (
-        <StyledIssueStats mode={mode} as="span" size="m" color={gray8}>
-            {nullable(issuers.length, () => (
-                <DotSep>
-                    <UserGroup users={issuers} />
-                </DotSep>
+        <div className={s.IssueStats}>
+            {nullable(owner, (o) => (
+                <Separator>
+                    <UserDropdown
+                        value={o}
+                        label="Owner"
+                        view="default"
+                        readOnly
+                        className={s.IssuesStatsUserDropdown}
+                    />
+                </Separator>
             ))}
 
-            {nullable(estimate, (e) => (
-                <DotSep>
-                    {formateEstimate(e, {
-                        type: estimateType ?? 'Strict',
-                        locale,
-                    })}
-                </DotSep>
+            {nullable(priority, (p) => (
+                <Separator>
+                    <PriorityDropdown label="Priority" value={p} view="default" readOnly />
+                </Separator>
             ))}
 
-            {nullable(priority, (p) => (
-                <DotSep>{getPriorityText(p)}</DotSep>
+            {nullable(estimate, (e) => (
+                <Separator>
+                    <EstimateDropdown
+                        value={{ date: getDateString(e), type: estimateType ?? 'Strict' }}
+                        label="Estimate"
```

Comment:
i18n?

Reply:
Prop `label` is typed. Based on the passed prop, text with i18n is selected in the dropdown